packages/cli/src/lionweb-cli.ts

```diff
@@ -69,7 +68,7 @@ If the chunk is the serialization of a LionCore Language/M2, then a textual rend
 `The ${GENERATE_TS_TYPES_COMMAND} command generates a TypeScript source files with type definitions for the given (JSON serializations of) languages, assuming the use of the dynamic façade.`
                 )
             } else {
-                generateTsTypesWith(commandArgs)
+                await generateTsTypesWith(commandArgs)
```

Comment:
Why made this async?

Reply:
`generateTsTypesWith` is already `async`, and `async` is viral :(